### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by TheLastProject**
Haven't tested the actual image yet, but this at least fixes the build.

### CODE REVIEW

**Comment by forslund** [comment]
Hi and many thanks!

Before we can merge this update we require you to sign the [Contributor License Agreement](https://mycroft.ai/CLA) to protect yourself and us from any legal issues.

After that I think this can be merged without problems. I would still like to pursuit the pip-install version since to me that feels closer to the production version and would be quite suitable for a container environment.

File: Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ RUN set -x \
 	&& cd /opt/mycroft \
 	&& mkdir /opt/mycroft/skills \
 	# git fetch && git checkout dev && \ this branch is now merged to master
-	&& /opt/mycroft/./dev_setup.sh --allow-root -sm \
+	&& yes | /opt/mycroft/./dev_setup.sh --allow-root -sm \
```
Comment (forslund) [inline Dockerfile]: There is a `CI` environment variable, when set it should bypass all yes/no questions could that be of use here?

**Comment by TheLastProject** [comment]
Filled in the request form, will sign as soon as I get a link.

>I would still like to pursuit the pip-install version since to me that feels closer to the production version and would be quite suitable for a container environment.

Sadly the Mycroft core package doesn't seem to have documented Python commands for everything and a fair bit of shell scripts in the repo, which makes the pip package by itself not enough to really do anything. But otherwise, I agree that'll be a good longer-term plan.

**Comment by forslund** [comment]
The CLA is a somewhat manual process so it could be one or two days before the email arrives.

File: Dockerfile (hunk 1 above)
Comment (TheLastProject) [reply Dockerfile]: Sadly not, AFAIK a docker build can't be interactive so this is needed regardless of if it's built on a CI or if someone runs `docker build` on their own machine

Comment (forslund) [reply Dockerfile]: I meant adding the env variable to this command line

`CI=true /opt/mycroft/./dev_setup.sh --allow-root -sm`

or similar

Comment (TheLastProject) [reply Dockerfile]: Ah, I didn't know about that variable existing, but looking at the code in mycroft-core it makes total sense. Changed it.

**Comment by krisgesling** [comment]
Thanks Sylvia, CLA confirmed and added to [contributors list](https://github.com/mycroftai/contributors)

**Pull request merged**